src/cleanlab_codex/validation.py

```diff
@@ -0,0 +1,107 @@
+    prompt_template: str = "Using the following Context, provide a helpful answer to the Query.\n\n Context:\n{context}\n\n Query: {query}",
+) -> bool:
+    """Check if a response is untrustworthy based on TLM's evaluation.
+
+    Uses TLM to evaluate whether a response is trustworthy given the context and query.
+    Returns True if TLM's trustworthiness score falls below the threshold, indicating
+    the response may be incorrect or unreliable.
+
+    Args:
+        response: The response to check from the assistant
+        context: The context information available for answering the query
+        query: The user's question or request
+        tlm: The TLM model to use for evaluation
+        threshold: Score threshold (0.0-1.0). Lower values allow less trustworthy responses.
+                  Default 0.6, meaning responses with scores less than 0.6 are considered untrustworthy.
+        prompt_template: Template for formatting the evaluation prompt. Must contain {context}
+                       and {query} placeholders.
+
+    Returns:
+        bool: True if the response is deemed untrustworthy by TLM, False otherwise
+    """
+    prompt = prompt_template.format(context=context, query=query)
+    resp = tlm.get_trustworthiness_score(prompt, response)
+    score: float = resp['trustworthiness_score']
+    return score < threshold
+
+# TLM Binary Classification
+def is_bad_response_unhelpful(response: str, tlm: TLM, query: Optional[str] = None, trustworthiness_score_threshold: Optional[float] = None) -> bool:
+    """Check if a response is unhelpful by asking TLM to evaluate it.
+
+    Uses TLM to evaluate whether a response is helpful by asking it to make a Yes/No judgment.
+    The evaluation considers both the TLM's binary classification of helpfulness and its
+    confidence score. Returns True only if TLM classifies the response as unhelpful AND
+    is sufficiently confident in that assessment (if a threshold is provided).
+
+    Args:
+        response: The response to check from the assistant
+        tlm: The TLM model to use for evaluation
+        query: Optional user query to provide context for evaluating helpfulness.
+              If provided, TLM will assess if the response helpfully answers this query.
+        trustworthiness_score_threshold: Optional confidence threshold (0.0-1.0).
+                                       If provided, responses are only marked unhelpful if TLM's
+                                       confidence score exceeds this threshold.
+
+    Returns:
+        bool: True if TLM determines the response is unhelpful with sufficient confidence,
+              False otherwise
+    """
+    if query is None:
+        prompt = (
+            "Consider the following AI Assistant Response.\n\n"
+            f"AI Assistant Response: {response}\n\n"
+            "Is the AI Assistant Response helpful? Remember that abstaining from responding is not helpful. Answer Yes/No only."
```

Comment:
```suggestion
            "Consider the following AI Assistant Response to a user.\n\n"
            f"AI Assistant Response: {response}\n\n"
            "Does the AI Assistant Response seem helpful? Things that are not helpful: abstaining from responding, or statements which are similar to 'I don't know', 'Sorry', or 'No information available'.  Answer Yes/No only."
```

Reply:
In testing, did you find asking "is helpful" more effective than asking "is unhelpful" ?

Reply:
I didn't notice a difference, but I've switched to asking to "is helpful".